src/planscape/forsys/forsys_request_params.py

```diff
@@ -349,7 +361,10 @@ def get_priority_weights_dict(self) -> dict[str, float]:
     def _read_url_params_with_defaults(self, params: QueryDict) -> None:
         _read_common_url_params(self, params)
         self.planning_area = get_default_planning_area()
-
+        self.max_area_per_project_in_km2 = self._read_positive_float(params,
```

Comment:
We need validation logic somewhere to ensure either max acreage or max cost is defined.

A few options:
1) add it to ForsysGenerationRequestParams and call it after setting values via ForsysGenerationRequestParamsFromUrlWithDefaults and 
It can be added to  it here, when ForsysGenerationRequestParamsFromDb.
2) add it to plan/views.py, where projects are created.
3) add it to both 1) and 2). 

Can you either add it to this pr, or write an issue for this?

Reply:
The request params now provide a default max area; it will never be undefined by the time we call forsys.  Both UrlWithDefaults and FromDb will use this default.

Reply:
sg - that works and it's simple.

I can see the default changing from 20 to something else as more e2e runs are made.

fyi: there's also a patchmax_proj_size_min, but I don't have any documented cases where it seemed to affect results, especially when used in conjunction with patchmax_proj_size.